app/qml/LayerAttributesPanel.qml

```diff
@@ -0,0 +1,32 @@
+import QtQuick 2.0
+import QtQuick.Controls 2.12
+
+Item {
+  id: root
+
+  property string selectedLayer: ""
+  signal backButtonClicked()
+
+  
+  Page {
+    id: attributesPage
+    anchors.fill: parent
+
+    header: PanelHeader {
+      id: attrPageHeader
+      height: InputStyle.rowHeightHeader
+      width: parent.width
+      color: InputStyle.clrPanelMain
+      rowHeight: InputStyle.rowHeightHeader
+      titleText: qsTr("Attributes")
+      
+      onBack: root.backButtonClicked()
+      withBackButton: true
+    }
+
+    Text {
+      anchors.centerIn: parent
+      text: qsTr("List of attributes for layer " + selectedLayer)
```

Comment:
`qsTr("List of attributes for layer %1").arg(selectedLayer)`

you cannot translate "selectedLayer" directly